openreview/conference/builder.py

```diff
@@ -819,6 +835,10 @@ def get_blind_submission_id(self, conference):
             name = 'Blind_' + name
         return conference.get_invitation_id(name)
 
+    def get_withdrawn_submission_id(self, conference):
+        name = 'Withdrawn_Submission'
```

Comment:
you could make this a default parameter:

`def get_withdrawn_submission_id(self, conference, name='Withdrawn_Submission'):`

Reply:
Agreed.